packit_service/utils.py

```diff
@@ -243,7 +243,7 @@ def pr_labels_match_configuration(
         f"(label.present: {configured_labels_present}, label.absent: {configured_labels_absent})"
     )
 
-    pr_labels = pull_request.labels
+    pr_labels = [label.name for label in pull_request.labels]
```

Comment:
Just to be sure, won't this break the use on GitLab/Pagure?

Reply:
oh, good point :face_exhaling: at least for GitLab these are indeed strings..I need to have a deeper look to fix it

Reply:
opened packit/ogr#839